course/templates/course/course_instances.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load course %}
+
+{% block title %}{% trans "Course instances" %} | {{ block.super }}{% endblock %}
+{% block view_tag %}instances{% endblock %}
+
+{% block content %}
+  <div class="frontpage-panel">
+    <div class="page-header">
+        <h1>{% trans "Course instances" %}</h1>
+    </div>
+    {% if instances %}
+      {% regroup instances by starting_time|date:"Y" as instances_by_year %}
+        {% for year, year_instances in instances_by_year %}
+          <section class="frontpage frontpage-section">
```

Comment:
`<section>` elements could have the attribute `aria-labelledby`. It can point to the `<h2>` with the year. No need to add any new string for this. It should be possible to make up unique ids for the h2 elements since there are many on the same page.